src/main/java/com/woowacamp/soolsool/core/cart/code/CartItemErrorCode.java

```diff
@@ -0,0 +1,22 @@
+package com.woowacamp.soolsool.core.cart.code;
+
+import com.woowacamp.soolsool.global.exception.ErrorCode;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum CartItemErrorCode implements ErrorCode {
```

Comment:
```suggestion
public enum CartErrorCode implements ErrorCode {
```

현재 패키지별로 ErrorCode를 구분하고 있어요. 근데 패키지 별로 구분하는 것도 복잡해서 클래스마다 구분해야할 것 같아요. 

`LiquorErrorCode` 보면 개행으로 구분하긴 했는데 복잡하쥬?

Reply:
우선 제안주신 코드대로 CartErrorCode로 명명했고, enum 원소들도 해당 오류코드를 던지는 코드마다 개행으로 구분했읍니다!